capabilities/replica/replica.ipldsch

```diff
@@ -0,0 +1,17 @@
+type Blob struct {
+  digest Multihash
+  size Int
+}
```

Comment:
Ideally we'd import this type from the blob schema in the `capabilities/blob/` package, however I don't believe IPLD schemas support import statements, thus it has been redefined here - open to ideas on improving this.

One alternative would be modifying the `capabilities/types` package to allow more than one schema to be imported via `LoadSchemaBytes` or similar.

Reply:
I agree giving `capabilities/types` the possibility of merging more than schema with the `types` base schema would be nice.

In this case, however, I'd just move the `Blob` type to the types schema, as I also redefined it in `space/blob` and could use it in `http/put` too.

Reply:
yea there's no import statements inside the schema. that said, if you look at the code for the LoadSchemaBytes function I wrote, there's certainly the ability to load a couple schemas together into a single type system. Adding the ability to accept multiple wouldn't be that hard. 

This was all never really finished over in go-ipld-prime land, so we work around it.

Reply:
also grand picture: I'd honestly be fine with just dumping the whole go-ipld-prime thing in favor of cbor-gen. go-ipld-prime is still a pain in the ass IMHO.

Reply:
> I'd honestly be fine with just dumping the whole go-ipld-prime thing in favor of cbor-gen

:fireworks: :heart: :pray: that would be stellar imo. I'd happily volunteer to do land that change when appropriate.